src/transport/raw/BUILD.gn

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/chip.gni")
+import("${chip_root}/src/ble/ble.gni")
+
+static_library("raw") {
+  output_name = "libRawTransport"
+
+  sources = [
+    "Base.h",
+    "MessageHeader.cpp",
+    "MessageHeader.h",
+    "PeerAddress.h",
+    "TCP.cpp",
+    "TCP.h",
+    "Tuple.h",
+    "UDP.cpp",
+    "UDP.h",
+  ]
+
+  public_deps = [
```

Comment:
Please add `-Wconversion` to cflags here; otherwise this effectively undoes part of https://github.com/project-chip/connectedhomeip/pull/2849

Reply:
Done. was able to do this only now as that CL is merged (otherwise we had compiler errors)

Reply:
Right, that's why I only asked for it now.  ;)